### PR TITLE
add clusterNetwork_CIDR and serviceNetwork variables

### PR DESCRIPTION
**Pull request by schabrolles**
Add the possibility to choose clusterNetwork and service Network CIDR.
(which is mandatory for implementing multicluster networking with RHACM and submariner)

### CODE REVIEW

**Review by Prajyot-Parab — APPROVED** [review]
/lgtm

File: playbooks/roles/ocp-config/defaults/main/main.yaml [hunk 1]
```diff
@@ -12,3 +12,6 @@ chronyconfig:
    enabled: true
 
 cni_network_provider: OpenshiftSDN
+
+clusterNetwork_CIDR: 10.128.0.0/14
+serviceNetwork: 172.30.0.0/16
```
Comment (Prajyot-Parab) [inline playbooks/roles/ocp-config/defaults/main/main.yaml]: Change var names to `cluster_network_cidr` and `service_network` (at all places)

**Review by Prajyot-Parab — CHANGES_REQUESTED** [review]

**Review by Prajyot-Parab — DISMISSED** [review]
/lgtm

File: playbooks/roles/ocp-config/defaults/main/main.yaml [hunk 2]
```diff
@@ -12,3 +12,6 @@ chronyconfig:
    enabled: true
 
 cni_network_provider: OpenshiftSDN
+
+cluster_network_cidr: 10.128.0.0/14
+service_network: 172.30.0.0/16
```
Comment (yussufsh) [inline playbooks/roles/ocp-config/defaults/main/main.yaml]: Could you please add a newline at the EOF?

**Review by yussufsh — APPROVED** [review]
/lgtm

**Review by bpradipt — APPROVED** [review]
/lgtm

**Pull request merged**